tmt.spec

```diff
@@ -32,34 +30,25 @@ metadata specification (L1 and L2) and allows easy test execution.
 
 %package -n     tmt+test-convert
 Summary:        Dependencies required for tmt test import and export
-Obsoletes:      tmt-test-convert < %{version}-%{release}
 Requires:       tmt == %{version}-%{release}
 Requires:       make
 Requires:       python3-bugzilla
 Requires:       python3-nitrate
 Requires:       python3-html2text
 Requires:       python3-markdown
 
-%description -n tmt+test-convert
-This is a metapackage bringing in extra dependencies for tmt.
-It contains no code, just makes sure the dependencies are installed.
+%description -n tmt+test-convert %_metapackage_description
 
 %package -n     tmt+provision-container
 Summary:        Dependencies required for tmt container provisioner
-Obsoletes:      tmt-provision-container < %{version}-%{release}
```

Comment:
Wouldn't it be a bit better/safer to keep these "dash" obsoletes and rather add corresponding `Provides`? It's possible that some automation still uses `tmt-provision-container` without the `+` character and this change would break their workflow. Similar for other obsoletes.

Reply:
Only epel9 would be affected by this. These obsoletes were added as par of `tmt 1.27.7`.

Last F37 build was `tmt 1.29` so the upgrade path to F39 is not affected. The first build in F39 that contains this is `tmt 1.28.2`, which looking at the timeline it was part of the initial release.

Epel9 is the only tricky one here. Maybe add a note about removing when epel9 is dropped? How far back are epel9 upgrade paths needed to be supported? I am mostly considering manual user installation. For packages it is rather easy to check with `fedrq`.

Reply:
@LecrisUT EPEL9 will be with us for quite some time :), but looking at https://packages.fedoraproject.org/pkgs/tmt/, epel9 do not have the "dash plugins", so I believe it's all good there.

@psss Your call, but from my PoV, 1.28 with the new packaging was released before F39 was, so I'm not sure how big is the chance that it would cause issues somewhere. When Fedora distro upgrade is being done, it first updates all packages, so F38 and F37 should be also safe, as there is 1.28 available.

Reply:
I was rather thinking about something like `tmt-provision-container` mentioned in an ansible playbook used to prepare the Testing Farm worker or something similar. And that would be broken regardless the upgrade path and Fedora version. Not sure on how many places the old package names can be still used. Probably not many but you know...

Reply:
Testing out https://github.com/teemtee/tmt/pull/2880/commits/6179d7eaad276f45ca7e41573822b70b4d1082f6. 
Was thinking about doing Obsolete foo < 1.28.0-1, but left it as is. We can remove all the obsoletes once we stop building for F39. Hopefully this will be enough to get rid of all the rpmlint issues.

Reply:
> I was rather thinking about something like `tmt-provision-container` mentioned in an ansible playbook used to prepare the Testing Farm worker or something similar.

That is a tricky one, but we can do a quick check in `sourcegraph`. It [looks fine](https://sourcegraph.com/search?q=context:global+tmt-provision-container&patternType=keyword&sm=0) but the other cases should also be checked. At the very least `testing-farm` needs to be checked.

>  We can remove all the obsoletes once we stop building for F39

On this issue I think it's fine because the the first build on F39 outside the beta period was `tmt 1.28.2` so it should already have had the `+` packages.

Reply:
I know, but they are still being available for installation, no? I'd just keep things safe for now and do additional cleanup after F42 is released. Wdyt?

Reply:
> other cases should also be checked.   

We'll keep the existing Provides regardless for the time being.

Reply:
> I know, but they are still being available for installation, no?

Ah good point. I did not check if the updated version is in base or updates repository.

> I'd just keep things safe for now and do additional cleanup after F42 is released. Wdyt?

:+1:. Safest option for now. Just having a comment to know when to revisit

Reply:
Now looks good, thanks!